Vagrantfile

```diff
@@ -16,14 +16,11 @@ Vagrant.configure("2") do |config|
   config.vm.box = "ubuntu/bionic64"
   config.vm.box_check_update = false
 
-  # TODO: Latest (2020-02-10)  ubuntu box is broken, force use of older box. Remove when upstream is fixed by vendor.
-  config.vm.box_url = "https://cloud-images.ubuntu.com/bionic/20200206/bionic-server-cloudimg-amd64-vagrant.box"
-
   config.vm.network "private_network", ip: "10.20.30.10"
 
   config.vm.provider "virtualbox" do |vb|
-    vb.memory = "4096"
-    vb.cpus = 4
+    vb.memory = "2096"
+    vb.cpus = 2
```

Comment:
I would recommend to leave memory and CPUs to what it was. Lowering resources will slow down build process for all.

Reply:
I should have not have to change the file, it was just practical, that on my laptop with 8GB I needed more VMs and needed lower limits to be able to work. Will try to get the file from master.

Reply:
I would recommend working remotely to the development server. It's faster than your laptop, and have plenty with memory :)

Reply:
I will now. Had problems with VPN before so used a local VM.